mozilla/bedrock
# Remove superfluous Glean metrics

## One-line summary

Since we now have standardized page_load and click events, none of this custom code is required any more 🎉 

## Issue / Bugzilla link

N/A

## Testing

1. Open http://localhost:8000/en-US/
2. Set `window.Glean.setLogPings(true)` in the web console and refresh the page.

[ ] You should no longer see the following metrics included in the page_load event:

```
    "datetime": {
      "page.viewed": "2024-06-17T10:13:29.392+01:00"
    },
    "string": {
      "page.path": "/",
      "page.locale": "en-US",
      "page.referrer": "",
      "page.http_status": "200"
    },
    "labeled_string": {
      "page.query_params": {
        "utm_source": "",
        "utm_campaign": "",
        "utm_medium": "",
        "utm_content": "",
        "entrypoint_experiment": "",
        "entrypoint_variation": "",
        "experiment": "",
        "variation": "",
        "v": "",
        "xv": ""
      }
    },
```
